.eslintignore

```diff
@@ -0,0 +1,2 @@
+/src/**/*.test.*
```

Comment:
Why do we want to ignore linting on tests?

Reply:
Good suggestion. I've fixed the lint warnings in test.